main/index.html

```diff
@@ -272,36 +218,18 @@ <h2>Features</h2>
         <div class="services-contents">
           <!-- Start Left services -->
           <div class="col-md-4 col-sm-4 col-xs-12">
-            <div class="about-move">
-              <div class="services-details">
-                <div class="single-services">
-                  <a class="services-icon" href="#">
-											<i class="fa fa-linux"></i>
-										</a>
-                  <h4>Recent software Linux add-on</h4>
-                  <p>
-                  MetWork contains a "dependencies package" (named <strong>mfext</strong>) ridiculously easy to install which add to a Linux distribution (including old ones) plenty of recent software, especially
-                                  for scientific and meteorology domain.
-                  </p>
-                  <a href="https://github.com/metwork-framework/mfext/blob/master/README.md#mfext" class="btn btn-info btn-xs" target="_blank">More details&nbsp;<span class="glyphicon glyphicon-share-alt" aria-hidden="true"></span></a>
-                </div>
-              </div>
-              <!-- end about-details -->
-            </div>
-          </div>
-          <div class="col-md-4 col-sm-4 col-xs-12">
-            <div class="about-move">
+            <!-- end col-md-4 -->
+            <div class=" about-move">
               <div class="services-details">
                 <div class="single-services">
                   <a class="services-icon" href="#">
-											<i class="fa fa-file"></i>
+											<i class="fa fa-cloud-download"></i>
 										</a>
-                  <h4>Incoming files processing framework</h4>
+                  <h4>Webservices framework</h4>
                   <p>
-                  With the "incoming files" processing framework (named <strong>mfdata</strong>), you can define a set of actions (you can provide as plugins) to easily build a custom but robust stream
-                                  processing workflow for your incoming files.
+                  Quickly build REST micro-services, websites and APIs with the <strong>mfserv</strong> module. It supports up-to-date technologies like "asynchronous Python3", nodejs and Python-Django
```

Comment:
```suggestion
                  Quickly build REST micro-services, websites and APIs with the <strong>mfserv</strong> module. It supports up-to-date technologies like "asynchronous Python3", nodejs, nginx/openresty
```

Reply:
Is there enough space to add nginx/openresty and Django?